src/components/log/Log.css

```diff
@@ -0,0 +1,45 @@
+.Log {
+  height: 50%;
+  width: 100%;
+  background-color: transparent;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.log-box {
+  height: 85%;
+  width: 90%;
+  color: white;
+  overflow-y: scroll;
+  text-align: left;
+  border-top: 1px solid grey;
+  background-color: rgba(75, 75, 75, 0.349);
+}
+
+@media only screen and (max-width: 650px) {
+  .Log {
+    margin-top: 20px;
+    height: 250px;
+  }
+}
+
+.log-level-wrap {
+  position: relative;
+  width: 90%;
+  margin-bottom: 5px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.log-level-wrap select {
```

Comment:
Apply some styling (e.g: font-size etc.) like I do in the current demo page to get rid of some of the ugly browser specific default styling and have this better match the rest of the design.